Makefile.am

```diff
@@ -53,6 +54,7 @@ swupd_make_fullfiles_SOURCES = \
 	src/delta.c \
 	src/fullfiles.c \
 	src/globals.c \
+	src/groups.c \
```

Comment:
Patrick and me dug into this, but with groups.c symbols used in manifest.c, we can't avoid this unless we shuffle a bunch of code around first. So this should be OK.